game/client_types/player.js

```diff
@@ -9,17 +9,17 @@
  * http://www.nodegame.org
  */
 
-var ngc = require('nodegame-client');
-var Stager = ngc.Stager;
-var stepRules = ngc.stepRules;
-var constants = ngc.constants;
+const ngc = require('nodegame-client');
+const Stager = ngc.Stager;
+const stepRules = ngc.stepRules;
+const constants = ngc.constants;
 
 // Export the game-creating function.
 module.exports = function(treatmentName, settings, stager, setup, gameRoom) {
 
     var cbs;
-    var channel = gameRoom.channel;
-    var node = gameRoom.node;
+    let channel = gameRoom.channel;
+    let node = gameRoom.node;
 
     // Import other functions used in the game.
     cbs = require(__dirname + '/includes/player.callbacks.js');
```

Comment:
Can you use `let` here, and also `path.join` instead of the string concatenation? The purpose of using `path.join` is to avoid to spell out the slashes, because they are platform dependent. In Windows they are `\` and in Linux/Max `/`.

Reply:
remember to require the `path` module above.

Reply:
ok, fixed!